src/main/java/org/cryptomator/cryptofs/DirectoryIdLoader.java

```diff
@@ -8,6 +8,7 @@
 import java.nio.channels.Channels;
 import java.nio.channels.FileChannel;
 import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
```

Comment:
unused import

Reply:
Fixed in [1f72a65](https://github.com/cryptomator/cryptofs/pull/118/commits/1f72a65ce639e8fe1fbd6e428bd6fb81268c0a1c)